### ISSUE TITLE
401 on ajax requests

### ISSUE THREAD

**Issue by engenb** [MohammadYounes/MVC5-MixedAuth#35]
Getting a cookie and loading my mvc site and resources just fine, but subsequent ajax requests are failing with a 401.  The cookie is in the request, but so is an Authorization: Negotiate header which I think is the cause of the 401 challenge.  Thoughts on this?

**Comment by MohammadYounes** [comment]
Try enabling windows authentication for the windows login end point `Windows/Login` ONLY (not the whole site)

**Comment by MohammadYounes** [comment]
Also are the ajax requests made to a WebApi or Static resources ? What browser are you using ? and is the App under IIS or IIS Express ?

**Comment by engenb** [comment]
sorry I forgot to post back.

the ajax requests were being made to a web api and I'm hosting my site in IIS.  I may look at enabling windows auth for just that endpoint, but for now things are happy.

I actually resolved my issue by registering my authorization middleware first, BEFORE calling app.UseWebApi(...).  I remembered that the order things get registered impacts how requests get processed.  I previously had my authorization middlware getting registered last, which didn't work.
